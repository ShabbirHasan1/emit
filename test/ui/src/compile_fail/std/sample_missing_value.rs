fn main() {
    emit::sample!(name: "my_metric");
}
