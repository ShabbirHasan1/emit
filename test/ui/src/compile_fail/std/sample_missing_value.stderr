error: the `value` parameter is required
 --> src/compile_fail/std/sample_missing_value.rs:2:5
  |
2 |     emit::sample!(name: "my_metric");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `emit::sample` (in Nightly builds, run with -Z macro-backtrace for more info)
